interop-testing/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
             project(':grpc-testing'),
             libraries.junit,
             libraries.mockito,
+            libraries.netty_tcnative,
```

Comment:
For now I've reverted this change (thus, the integration tests run with Jetty ALPN only, not with OpenSSL), although personally I think there really ought to be integration tests that use the recommended configuration.